yb-voyager/src/pgss/collector.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package pgss
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+// CollectFromPostgreSQL collects PGSS data directly from a PostgreSQL database or assessmentDB
```

Comment:
I suspect this will look a little different later on. Ignoring this.

Reply:
yeah i will remove this one.